recipes/drmaa/run_test.py

```diff
@@ -0,0 +1,4 @@
+try:
+    import drmaa
```

Comment:
Even if we include libdrmaa, it still won't quite work. This would require a fully configured and working SGE Cluster install.

We could do this for testing with the package manager. We could also use a docker container like this [one]( https://github.com/jakirkham/docker_centos_drmaa_conda ).

Though I agree with @dan-blanchard that taking SGE into `conda` feels like quite a bit of work. I looked at it once, but getting it built is intimidating to say the least and requires a host of dependencies that we don't have yet. Not to mention, it probably won't be configured correctly (or maybe work?) for most use cases.

A better solution might be to just add some notes here explaining clearly how this must be configured in order to get it to work. We could even have this printed during install using a pre or post link script.

Thoughts?